Source/WTF/Scripts/Preferences/WebPreferencesInternal.yaml

```diff
@@ -1019,11 +1019,12 @@ ShouldTakeSuspendedAssertions:
   type: bool
   humanReadableName: "Take WebKit:Suspended assertions on background web content processes"
   humanReadableDescription: "Take WebKit:Suspended assertions on background web content processes"
+  exposed: [ WebKit ]
   defaultValue:
     WebKitLegacy:
```

Comment:
Since you used `exposed: [ WebKit ]`, you don't need to provide a default value for WebKitLegacy or WebCore I believe.

Reply:
I was getting build issues when I tried this so I'm just going to merge as is if that's alright